NIX_Lessons/lesson10/src/main/java/nix/lessons/Cat.java

```diff
@@ -0,0 +1,72 @@
+package nix.lessons;
+
+
+import java.util.Random;
+
+public class Cat extends Predator{
```

Comment:
nit: I'd recommend using the auto-formatting shortcut in Intellij https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html

Reply:
I am sorry but I don't understand what you mean in this case.  Could you explain what is wrong in this fragment or file?